lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -886,12 +901,14 @@ def commit(order, message_type, retry_logic = nil, trace_number = nil)
             request.call(remote_url(:secondary))
           end
 
+        authorization = order.include?('<TokenTxnType>GT</TokenTxnType>') ? response[:safetech_token] : authorization_string(response[:tx_ref_num], response[:order_id])
```

Comment:
@gasb150 and @edgarv09's questions got me thinking about this - should we just append the `safetech_token` to the `authorization_string`? I'm thinking something like what we do in [Adyen](https://github.com/activemerchant/active_merchant/blob/6580e97c12b8b629303656e42c70c0d6d799808e/lib/active_merchant/billing/gateways/adyen.rb#L855-L863).

If the gateway is returning `tx_ref_num` and `order_id`, I feel we should capture those values and append the `safetech_token` to the `;` delimited string, then parse it out when we need it.

Reply:
Hi @naashton I think this is a special case, because of authorization value should match with the `safetech_token` stored by Orbital, I mean, when we send an authorization/purchase the `safetech_token` value is sent int the `cardNumber` field, then if we use this token along with `tx_ref_num` and `order_id`, we would get the next error.
`"Error. The Orbital Gateway has received a badly formatted message. Field Card Number exceeded max length of 19"`